pkg/loghelper/loghelper.go

```diff
@@ -34,3 +34,7 @@ func (list CuratedLogLines) AddLogLine(format string, args ...interface{}) Curat
 	logrus.Debug(message)
 	return list
 }
+func (list CuratedLogLines) Append(newLines CuratedLogLines) CuratedLogLines {
```

Comment:
This function shouldn't return `CuratedLogLines`.  There should be a getter or something to retrieve the appended lines.

Reply:
Returning `list` seems like it's returning a copy of the original object since its not a pointer.

Reply:
yeah correct. looks like I did not test this function. Since it is not used, I will remove it for now